src/component/auth_providers/mod.rs

```diff
@@ -132,9 +155,31 @@ pub fn LoginProviders(show_modal: RwSignal<bool>, lock_closing: RwSignal<bool>)
             // This is some redundant work, but saves us 100+ lines of resource handling
             let canisters = do_canister_auth(Some(identity.clone())).await?.unwrap();
 
-            if let Err(e) = handle_user_login(canisters, referrer).await {
+            if let Err(e) = handle_user_login(canisters.clone(), referrer).await {
                 log::warn!("failed to handle user login, err {e}. skipping");
             }
+
+            #[cfg(all(feature = "hydrate", feature = "ga4"))]
```

Comment:
i would much rather have this behind an abstraction to avoid using the cfg macro everytime

Reply:
will do post release